build/iOS/deps/irrlicht.sh

```diff
@@ -3,12 +3,12 @@
 . sdk.sh
 
 [ ! -d irrlicht-src ] && \
-	svn co -r 5415 svn://svn.code.sf.net/p/irrlicht/code/branches/ogl-es irrlicht-src
+	svn co -r 5573 svn://svn.code.sf.net/p/irrlicht/code/branches/ogl-es irrlicht-src
```

Comment:
Why exactly 5573? Any problems with https://sourceforge.net/p/irrlicht/code/5597/

Reply:
Didn't see that there was a newer revision, you can update it to 5597.